docs/_includes/Modules.md

```diff
@@ -129,29 +135,42 @@ module Mod {
 Note that inside `m()`, we have to use `A` instead of `Helpers`, as we bound
 it to a different name. The name `Helpers` is not available inside `m()`,
 as only names that have been bound inside `Mod` are available. In order
-to use the members from another module, it either has to be declared
+to use the members from another module, that other module either has to be declared
 there with `module` or imported with `import`.
 
 We don't have to give `Helpers` a new name, though, if we don't want
-to. We can write `import Helpers = Helpers` if we want to, and Dafny
+to. We can write `import Helpers = Helpers` to import the module under
+its own name; Dafny
 even provides the shorthand `import Helpers` for this behavior. You
 can't bind two modules with the same name at the same time, so
 sometimes you have to use the = version to ensure the names do not
-clash.
+clash. When importing nested modules, `import B.C` means `import C = B.C`;
+the new name is always the last name segment of the module designation.
 
 The ``QualifiedModuleName`` in the ``ModuleImport_`` starts with a
 sibling module of the importing module, or with a submodule of the
-importing module. There is no wya to refer to the parent module, only
+importing module. There is no way to refer to the parent module, only
 sibling modules (and their submodules).
 
+Import statements may occur at the top-level of a file as well.
```

Comment:
Although the declarations of a program are divided into files, I think of a Dafny program as a set of modules, not a set of files. Therefore, instead of "at the top-level of a file", I'd prefer one of:
* at the top-level
* in the top-level module
* in the implicit top-level module
* in the (implicit) top-level module

or something like that. The text below just says "at the top-level", which seems fine to me.

Reply:
Fixed